### DISCUSSION TITLE
Explanation of some features

### DISCUSSION THREAD

**Discussion by Flylow260** [YimMenu/YimMenu#3236] [Q&A]
I have been using Yim for a little while now, and I really like it. The creators are doing a great job. However, my problem is that there are only a few videos explaining certain functions. I have noticed some features that I don't understand and can't find information about online. The options that interest me the most are "Stat Editor" under the Network tab, "Change State Immediately" in the vehicle tab, and "Warp Time" in the toxic options for players. I would be grateful if someone could explain these to me.
![1](https://github.com/YimMenu/YimMenu/assets/166236811/fe157e92-62cc-45f3-9120-e8fe18865509)
![2](https://github.com/YimMenu/YimMenu/assets/166236811/2e10088c-a609-48fe-a086-e282f7512fb5)
![3](https://github.com/YimMenu/YimMenu/assets/166236811/f72d8a05-9b11-4b56-ad4c-c3ec7c771b1b)

**Comment by D-Borkan** [comment]
state editor work like "GTAHaX" but with more and advanced options
warp time it's change the time  and work only when you be a host

**Reply by Flylow260** [reply]
Thanks for the explanation but for better understanding, so I can use the gtahax' stats and values and the warp time is just a "time travel" in real life time but in-game?

**Reply by D-Borkan** [reply]
yes time the game
